### DISCUSSION TITLE
`npm ci` hangs in actions on MacOS right now

### DISCUSSION THREAD

**Discussion by barmac** [community/community#112433] [npm]
### Select Topic Area

Bug

### Body

In camunda/camunda-modeler, we have been experiencing job hanging on `npm ci` on MacOS from yesterday on: https://github.com/camunda/camunda-modeler/actions/runs/8269463124/job/22652653794
Not sure if this is on GH Actions or npm.

**Comment by saeedhosan** [comment]
npm ci is clean install package by package-lock.json 
make sure you have a package-lock.json file when you run npm ci 

Also you may test on locally by running npm ci that works without warning

**Comment by barmac** [comment]
We are no more experiencing this issue.
